lib/afc/index.js

```diff
@@ -314,15 +314,17 @@ class AfcService extends BaseServiceSocket {
   _createPacketPromise (message, timeout = 10000) {
     const packetNumber = this._packetNumber++;
     const response = new B((resolve, reject) => {
-      this._responseCallbacks[packetNumber] = (data) => resolve(data);
-      setTimeout(() => reject(new Error(`Cound't finish the operation "${message}". Failed to receive any data within the timeout: ${timeout}`)), timeout);
+      this._responseCallbacks[packetNumber] = resolve;
+      setTimeout(function () {
+        reject(new Error(`Could not finish the operation '${message}'. Failed to receive any data within the timeout: ${timeout}`));
```

Comment:
I's rather change the description to `Failed to receive any data within ${timeous}ms timeout`